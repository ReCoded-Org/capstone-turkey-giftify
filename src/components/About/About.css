.aboutUsMainDiv{
    background-color: #0077B6;
    display: flex;
	flex-direction: column;
	flex-wrap: nowrap;
	justify-content: flex-start;
	align-items: center;
	align-content: stretch;
    font-family: "Roboto", sans-serif;
}

.upperPartAbout{
	display: flex;
	flex-direction: column;
	flex-wrap: nowrap;
	justify-content: space-between;
	align-items: center;
	align-content: stretch;
}

.upperPartAbout > h1{
    font-style: normal;
    font-weight: normal;
    font-size: 70px;
    line-height: 82px;
    display: flex;
    align-items: center;
    text-align: center;
    color: #FFFFFF;
}

.visionMisionMainDiv{
    background-color: #CAF0F8;
    border-radius: 20px;
    text-align: center;
    display: flex;
	flex-direction: column;
	flex-wrap: nowrap;
	justify-content: flex-start;
	align-items: stretch;
	align-content: stretch;
}

.middleArea{
    display: flex;
	flex-direction: row;
	flex-wrap: nowrap;
	justify-content: center;
	align-items: center;
	align-content: stretch;
    width: 100%
}

.middleAreaTextbox{
    width: 65%;
    background-color: #CAF0F8;
    border-radius: 20px;
}

.middleAreaTextbox > p{
    margin: 5%;
}

.middleAreaImage{
    width: 35%;
    
}

.middleAreaImage > img{
    max-width: 200px;
    border-radius: 33px;
}

@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");