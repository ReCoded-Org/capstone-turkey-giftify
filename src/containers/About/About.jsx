import React from "react";
import AboutComponent from "./../../components/About/AboutComponent";

const About = () => {
  return <AboutComponent />;
};

export default About;
