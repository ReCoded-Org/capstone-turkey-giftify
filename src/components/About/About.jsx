import React from "react";
import "./About.css";
import aboutImage from "./aboutImage.png";
import { Row } from "react-bootstrap";

const About = () => {
  return (
    <div className="aboutUsMainDiv">
      <Row className="upperPartAbout">
        <h1>About Us</h1>
        <div className="visionMisionMainDiv">
          <p>
            <span className="spanDescription">Our Vision: </span>
            Everyone can get what they want in a fast and easy way
          </p>
          <p>
            <span className="spanDescription">
              <br />
              Our Mision:
            </span>
            Trade things are no longer used between people for a clean
            environment
          </p>
        </div>
      </Row>
      <Row className="middleArea">
        <div className="middleAreaTextbox">
          <p>
            Giftify is a platform for trading things that are no longer used
            between people who stay in the same area. People have the chance to
            get their needs in free by giving the convincing reason of needing a
            specific product/service to the owner then the owner has the freedom
            to choose which applicant has deserved his product or service more
            than the other applicants.
          </p>
        </div>
        <div className="middleAreaImage">
          <img src={aboutImage} alt="About pic" />
        </div>
      </Row>
    </div>
  );
};

export default About;
