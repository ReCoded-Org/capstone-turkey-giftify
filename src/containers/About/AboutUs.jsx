import React from "react";
import About from "./../../components/About/About";
import OurTeam from "./../../components/OurTeam/OurTeam";

const AboutUs = () => {
  return (
    <>
      <About />
      <OurTeam />
    </>
  );
};

export default AboutUs;
