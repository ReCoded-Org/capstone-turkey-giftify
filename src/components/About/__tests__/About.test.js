import React from "react";
import About from "./../About";
import { cleanup } from "@testing-library/react";
import renderer from "react-test-renderer";

afterEach(cleanup);

it("matches snapshot", () => {
  const tree = renderer.create(<About />).toJSON();
  expect(tree).toMatchSnapshot();
});
