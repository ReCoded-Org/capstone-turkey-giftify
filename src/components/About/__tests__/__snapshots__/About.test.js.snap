// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About rendered correctly 1`] = `
<div
  className="aboutUsMainDiv"
>
  <div
    className="upperPartAbout row"
  >
    <h1>
      About Us
    </h1>
    <div
      className="visionMisionMainDiv"
    >
      <p>
        <span
          className="spanDescription"
        >
          Our Vision: 
        </span>
        Everyone can get what they want in a fast and easy way
      </p>
      <p>
        <span
          className="spanDescription"
        >
          <br />
          Our Mision:
           
        </span>
        Trade things are no longer used between people for a clean environment
      </p>
    </div>
  </div>
  <div
    className="middleArea row"
  >
    <div
      className="middleAreaTextbox"
    >
      <p>
        Giftify is a platform for trading things that are no longer used between people who stay in the same area. People have the chance to get their needs in free by giving the convincing reason of needing a specific product/service to the owner then the owner has the freedom to choose which applicant has deserved his product or service more than the other applicants.
      </p>
    </div>
    <div
      className="middleAreaImage"
    >
      <img
        alt="About pic"
        src="aboutImage.png"
      />
    </div>
  </div>
</div>
`;
